bin/message_queue_daemon

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+require './config/environment.rb'
+
+adapter = Class.new do
+  def configure_start_command(command)
+    command.option :verbose, '-v', '--verbose', 'Be verbose.'
+  end
+
+  def on_start(options, helper)
+    puts "on_start #{options.inspect}"
+  end
+
+  def configure_supervisor(supervisor)
+    options = {
+      name: 'hermes.queue',
+      :path => '**',
+      :klass => 'post.hermes_message',
+      :event => 'create',
```

Comment:
I agree this is a potential hazard, but wouldn't this be a design flaw under which all apps using queue listeners suffer? How could we reliably listen only to data changes which where initiated by a specific pebble?